
This patch reverses video profiles order in V4L2 Codecs profile.


Past:
priority1: ffmpeg/opengl-yuv
priority2: v4l2

Now:
priority1: v4l2
priority2: ffmpeg/opengl-yuv


diff --speed-large-files --no-dereference --minimal -Naur mythtv-53dbe79f11fe2baafacf74559ac8ef5904f08bee/mythtv/libs/libmythtv/mythvideoprofile.cpp mythtv-53dbe79f11fe2baafacf74559ac8ef5904f08bee/mythtv/libs/libmythtv/mythvideoprofile.cpp
--- mythtv-53dbe79f11fe2baafacf74559ac8ef5904f08bee/mythtv/libs/libmythtv/mythvideoprofile.cpp	2022-11-01 21:24:15.000000000 +0100
+++ mythtv-53dbe79f11fe2baafacf74559ac8ef5904f08bee/mythtv/libs/libmythtv/mythvideoprofile.cpp	2022-11-02 12:34:39.913102476 +0100
@@ -1194,7 +1194,7 @@
     {
         (void)tr("V4L2 Codecs", "Sample: V4L2");
         uint groupid = CreateProfileGroup("V4L2 Codecs", HostName);
-        CreateProfile(groupid, 2, "", "", "",
+        CreateProfile(groupid, 1, "", "", "",
                       "v4l2", 1, true, "opengl-hw",
                       "shader:driver:medium", "shader:driver:medium");
         CreateProfile(groupid, 1, "", "", "",
